v3/setup/images-control.sh

```diff
@@ -12,12 +12,13 @@ if [ "$(etcdctl get /bootstrap.service/images-control-bootstrapped)" == "true" ]
 fi
 etcdctl set /bootstrap.service/images-control-bootstrapped true
 
-etcdctl set /images/chronos      	"mesosphere/chronos:chronos-2.4.0-0.1.20150828104228.ubuntu1404-mesos-0.27.0-0.2.190.ubuntu1404"
-etcdctl set /images/flight-director "behance/flight-director:latest"
-etcdctl set /images/marathon     	"mesosphere/marathon:v0.15.1"
-etcdctl set /images/mesos-master 	"mesosphere/mesos-master:0.27.0-0.2.190.ubuntu1404"
-etcdctl set /images/zk-exhibitor 	"behance/docker-zk-exhibitor:latest"
-etcdctl set /images/cfn-signal   	"behance/docker-cfn-bootstrap:latest"
+etcdctl set /images/chronos                   "mesosphere/chronos:chronos-2.4.0-0.1.20150828104228.ubuntu1404-mesos-0.27.0-0.2.190.ubuntu1404"
+etcdctl set /images/flight-director           "behance/flight-director:latest"
+etcdctl set /images/marathon                  "mesosphere/marathon:v0.15.1"
+etcdctl set /images/mesos-master              "mesosphere/mesos-master:0.27.0-0.2.190.ubuntu1404"
+etcdctl set /images/zk-exhibitor              "behance/docker-zk-exhibitor:latest"
+etcdctl set /images/cfn-signal                "behance/docker-cfn-bootstrap:latest"
+etcdctl set /images/booster/metrics-producer  "behance/booster:fdcloudwatch-latest"
```

Comment:
Is this a typo? `behance/booster:fdcloudwatch-latest`? Typically the tag name is `:latest` not `-latest`.